omniauth-indieauth.gemspec

```diff
@@ -17,6 +17,10 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- exe/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'faraday', '~> 0.9'
+  s.required_ruby_version = '>= 2.5'
```

Comment:
A newer Bundler convention that I’ve seen is:

```rb
spec.required_ruby_version = Gem::Requirement.new('>= 2.5', '< 2.8')
```
Note, too, the addition of the upper bound to handle if/when a newer Ruby is released and this gem is untested against it.

Reply:
Great catch, especially since I'm pinning this between 2.5 and 2.7 in terms of Ruby version support.